main.py

```diff
@@ -282,6 +283,67 @@ async def on_message_edit(before, after):
             ctx = await bot.get_context(after)
             await bot.invoke(ctx)
 
+@bot.event
+async def on_message(message):
+	if bot.afk_users.get(message.author.id):
+			del bot.afk_users[message.author.id]
+			return await message.channel.send(f'Welcome back {message.author.name}, you are no longer AFK')
+
+	for mention in message.mentions:
```

Comment:
There's more we can do with this. It would be nice to have it say "person1 and person2 are afk" instead of just sending one person's message.